### ISSUE TITLE
FEAT: Right-justify columns containing number data types

### ISSUE THREAD

**Issue by anthonydb** [beekeeper-studio/beekeeper-studio#424]
Query results in Beekeeper show number columns as left-justified. This makes comparing values harder; it's also different from psql behavior.

Beekeeper:

![Screen Shot 2020-10-30 at 9 36 17 AM](https://user-images.githubusercontent.com/273891/97711944-213c1600-1a94-11eb-9dfe-afa5249ef252.png)

PSQL:

![Screen Shot 2020-10-30 at 9 40 25 AM](https://user-images.githubusercontent.com/273891/97711965-2731f700-1a94-11eb-9a6d-8c9f229fe900.png)

Request changing the default display for those types.

**Comment by rathboma** [comment]
That's true, and would align with excel. Mayve they can also be formatted (, / .).

@ghstng wanna look at this one too? :-) Not sure if this should be in the 1.9 milestone?

**Comment by anthonydb** [comment]
After the 5.0 announcement, I tried Beekeeper after being away for a while. This issue is one that seems like it would be easy to remedy and would make the results grid more usable, in my opinion.

**Comment by anthonydb** [comment]
Hi, just installed version 5.3.4 to look at how this is working. This is what I see on two numeric columns:

<img width="299" height="341" alt="Image" src="https://github.com/user-attachments/assets/38f8254f-8799-4e99-b634-28b242a6de0f" />

Connected to PostgreSQL. Is that expected?
